Sources/protoc-gen-swift/ExtensionGenerator.swift

```diff
@@ -115,10 +115,13 @@ struct ExtensionGenerator {
         let scope = swiftDeclaringMessageName == nil ? "" : "static "
         let traitsType = descriptor.getTraitsType(context: context)
 
-        var fieldNamePath = self.protoPath
-        if fieldNamePath.hasPrefix(".") {
-            fieldNamePath.remove(at: fieldNamePath.startIndex)
-            fieldNamePath = "\"\(fieldNamePath)\""
```

Comment:
I put the quotes into the `p.print` below.

I also moved the calculation of the fully-scoped field name into the initializer.